usermod: update Spanish translation

ikks opened the pull request:
usermod: update Spanish translation

- [X ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ X] The page description(s) have links to documentation or a homepage.
- [ X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ X] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [ X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

kant commented on pages.es/linux/usermod.md:
```diff
@@ -1,12 +1,21 @@
 # usermod
 
 > Modifica una cuenta de usuario.
+> Ver también: `users`, `useradd`, `userdel`.
```
```suggestion
> Vea también: `users`, `useradd`, `userdel`.
```

ikks replied:
In other translations it seemed that imperative was avoided.  Maybe we can continue with this line of thought?

El imperativo es un poco más regional, cuando se usa con usted o con tu, por ejemplo vea o ve (ambos del verbo ver). Si lo hacemos en impersonal, tal vez es menos regionalizado en la preferencia al tutear. Sin embargo, puede ser una buena idea tener unos lineamientos en algún sitio específico para español,  tal vez me faltó mirar un poco más de documentación y seguir lo que ya se ha hecho previamente.  

Yo continúo como venga la línea :)

kant commented on pages.es/linux/usermod.md:
```diff
@@ -1,12 +1,21 @@
 
+- Cambia el ID de un usuario:
+
+`sudo usermod --uid {{id}} {{username}}`
```
```suggestion
`sudo usermod --uid {{id}} {{nombre_usuario}}`
```

kant commented on pages.es/linux/usermod.md:
```diff
@@ -1,12 +1,21 @@
+
+`sudo usermod --uid {{id}} {{username}}`
+
+- Cambia la interfaz de comandos(shell) a un usuario:
```
```suggestion
- Cambia la interfaz de comandos (shell) a un usuario:
```

kant commented on pages.es/linux/usermod.md:
```diff
@@ -1,12 +1,21 @@
+
+- Cambia la interfaz de comandos(shell) a un usuario:
+
+`sudo usermod --shell {{path/to/shell}} {{username}}`
```
```suggestion
`sudo usermod --shell {{ruta/a/interfaz_comando}} {{nombre_usuario}}`
```

kant commented on pages.es/linux/usermod.md:
```diff
@@ -1,12 +1,21 @@
+
+`sudo usermod --shell {{path/to/shell}} {{username}}`
+
 - Añade un usuario a grupos suplementarios (tener en cuenta los espacios en blanco):
```
```suggestion
- Añade un usuario a grupos suplementarios (ten en cuenta los espacios en blanco):
```

ikks replied:
Related to a previous comment, let me know if we use "tenga" or "ten" to sync.

kant replied:
ten

ikks commented:
Seems like a false positive what tldr-bot is suggesting
![Screenshot from 2024-10-24 10-41-23](https://github.com/user-attachments/assets/0cccb20e-2464-4f0c-a78b-23acf0a1db6f)

kant commented:
> Seems like a false positive what tldr-bot is suggesting ![Screenshot from 2024-10-24 10-41-23](https://private-user-images.githubusercontent.com/531285/379841673-0cccb20e-2464-4f0c-a78b-23acf0a1db6f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk3ODcyMTgsIm5iZiI6MTcyOTc4NjkxOCwicGF0aCI6Ii81MzEyODUvMzc5ODQxNjczLTBjY2NiMjBlLTI0NjQtNGYwYy1hNzhiLTIzYWNmMGExZGI2Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyNFQxNjIxNThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jNzUzZDgzNWQxMmJiYjg4N2M0MDI5MDYwMjkzM2IzMDU5ZjJhODU3NzMyODM5YWViNTU0MmYwNWFhNmEwMDA2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.wc_bVY1HD9wT3-9brtDkgkHBa05R5jkb4Wd6XEkn72Q)

There is some changes in commands, look closely

ikks commented:
> 
> There is some changes in commands, look closely

You are right, thanks for spotting it, I fixed my upstream reference. Just updated, going for the others.

ikks commented:
> Please fix the tldr-bot comments.

Addressed, thank you @sebastiaanspeck

ikks commented:
Please some help here, I'm convinced we are up to date
```i@h:tldr$ pr -t -w 140 -m pages/linux/usermod.md pages.es/linux/usermod.md 
# usermod							     # usermod
								     
> Modify a user account.					     > Modifica una cuenta de usuario.
> See also: `users`, `useradd`, `userdel`.			     > Vea también: `users`, `useradd`, `userdel`.
> More information: <https://manned.org/usermod>.		     > Más información: <https://manned.org/usermod>.
								     
- Change a username:						     - Cambia el nombre de un usuario:
								     
`sudo usermod {{-l|--login}} {{new_username}} {{username}}`	     `sudo usermod {{-l|--login}} {{nuevo_nombre}} {{usuario}}`
								     
- Change a user ID:						     - Cambia el ID de un usuario:
								     
`sudo usermod {{-u|--uid}} {{id}} {{username}}`			     `sudo usermod {{-u|--uid}} {{id}} {{usuario}}`
								     
- Change a user shell:						     - Cambia la interfaz de comandos (shell) a un usuario:
								     
`sudo usermod {{-s|--shell}} {{path/to/shell}} {{username}}`	     `sudo usermod {{-s|--shell}} {{ruta/a/interfaz_comando}} {{usuario}}`
								     
- Add a user to supplementary groups (mind the lack of whitespace):   - Añade un usuario a grupos suplementarios (ten en cuenta los espacios
								     
`sudo usermod {{-a|--append}} {{-G|--groups}} {{group1,group2,...}} { `sudo usermod {{-a|--append}} {{-G|--groups}}  {{grupo1,grupo2}} {{us
								     
- Change a user home directory:					     - Cambia el directorio home de un usuario:
								     
`sudo usermod {{-m|--move-home}} {{-d|--home}} {{path/to/new_home}} { `sudo usermod {{-m|--move-home}} {{-d|--home}} {{ruta/al/nuevo_home}}
```
